.vscode/settings.json

```diff
@@ -129,5 +129,35 @@
         "yaml.schemas": {
                 "${workspaceFolder}/validation/module_schema.yaml": "${workspaceFolder}/src/modules/*/module.yaml"
         },
-        "ros.distro": "humble"
+        "ros.distro": "humble",
+        "python.autoComplete.extraPaths": [
+                "/home/roque/gz_garden_ws/install/local/lib/python3.10/dist-packages",
```

Comment:
roque?

Reply:
fixed!

PS: I really need to make sure that my .vscode is always ignored for committing in PX4